homeassistant/components/media_source/local_source.py

```diff
@@ -118,7 +149,7 @@ def _build_item_response(self, source_dir_id: str, path: Path, is_child=False):
 
         media = BrowseMediaSource(
             domain=DOMAIN,
-            identifier=f"{source_dir_id}/{path.relative_to(self.hass.config.path('media'))}",
+            identifier=f"local_source/{source_dir_id}/{path.relative_to(self.hass.config.media_dirs[source_dir_id])}",
```

Comment:
There is no reason to add local_source, since we have `DOMAIN` already to identify ?